extension/Hello/src/Task/HelloStagedTaskSet.php

```diff
@@ -434,6 +434,38 @@ public function getSuccessor(): ?string
         return null;
     }
 
+    /**
+     * @return array<string, array<string>>
```

Comment:
Please, update method's docblock according to our conventions https://github.com/spryker-sdk/sdk/blob/master/docs/conventions.md#contracts